server/server.js

```diff
@@ -1,23 +1,24 @@
 const express = require('express');
-let app = express();
-let { retrieve, seeder } = require('./db/index.js')
+const app = express();
+const { retrieve } = require('./db/index.js')
+const cors = require('cors')
+const bodyParser = require ('body-parser');
+require('dotenv').config()
 
 app.use(express.static(__dirname + '/../client/dist'));
+app.use(cors())
 
-app.get('/', function (req, res) {
-  console.log('Got request');
-  seeder();
-  res.end();
+app.get('/amenities-api/amenities', function (req, res) {
+  retrieve({})
+    .then(data => res.json(data));
 });
 
-app.get('/show', function (req, res) {
-  retrieve({}) 
-    .then(data => {
-      console.log('Getting all data: ', data.length);
-      res.json(data);
-    });
+app.get('/amenities-api/amenity/:id', function (req, res) {
+  const { id } = req.params;
+  retrieve({ id })
+    .then(data => res.json(data[0]));
```

Comment:
This line returning `data[0`] rather than `data` created a small break in the nearby-buildings component